Makefile

```diff
@@ -15,7 +15,7 @@ MAKEFLAGS += --warn-undefined-variables
 
 # This version-strategy uses git tags to set the version string
 VERSION ?= $(shell git describe --tags --always --dirty)
-VERSION ?= v0.0.1
+VERSION ?= v0.0.1unknown
```

Comment:
Just double checking if this is intentional

Reply:
It is. I added this suffix to denote this is an invalid tag.